app/screens/home/search/initial/modifiers/index.tsx

```diff
@@ -4,99 +4,98 @@
 import React, {type Dispatch, type RefObject, type SetStateAction, useCallback, useEffect, useMemo, useRef, useState} from 'react';
 import {type IntlShape, useIntl} from 'react-intl';
 import {View} from 'react-native';
-import Animated, {useSharedValue, useAnimatedStyle, withTiming} from 'react-native-reanimated';
+import Animated, {type SharedValue, useSharedValue, useAnimatedStyle, withTiming} from 'react-native-reanimated';
 
 import FormattedText from '@components/formatted_text';
 import {useTheme} from '@context/theme';
-import TeamPickerIcon from '@screens/home/search/team_picker_icon';
+import TeamPicker from '@screens/home/search/team_picker';
 import {makeStyleSheetFromTheme} from '@utils/theme';
 import {typography} from '@utils/typography';
 
+import {ALL_TEAMS_ID} from '../..';
```

Comment:
not that it matters as this will be moved to a constants file, but please do avoid imports from `../` and `../..`etc..

Reply:
Fixed